docs/solrini.md

```diff
@@ -98,6 +99,54 @@ Make sure `core18` collection is created and `/path/to/WashingtonPost` is update
 Solrini has also been verified to work with the [MS MARCO Passage Retrieval Corpus](experiments-msmarco-passage.md).
 There should be no major issues with other collections that are supported by Anserini, but we have not tested them.
 
+## Solr with Prebuilt Lucene Index
+
+Solr can be considered a front-end for Lucene, and it is entirely possible for Solr to read prebuilt Lucene indexes. 
+To achieve this, some housekeeping are required. The following uses [robust04](regressions-robust04.md) as an example.
+
+First, a Solr collection must be created to house the index. Here we create a collection `robust04` with configset `anserini`.
+
+```
+solrini/bin/solr create -n anserini -c robust04
+```
+
+Along with the collection, Solr will create a core instance, whose name can be found in the Solr UI under collection overview.
+Which might look something like `<collection_name>_shard<id>_replica_<id>` (e.g., `robust04_shard1_replica_n1`). Solr stores configurations and data for the core instances under Solr home, which for us is `solrini/server/solr/` by default.
+
+Second, make proper Solr schema adjustments if required.
+Here `robust04` is a Trec collection whose schema is already taken care of by [managed-schema](https://github.com/castorini/anserini/blob/master/src/main/resources/solr/anserini/conf/managed-schema) in the Solr configset.
+However, if you are dealing with a collection such as `cord19`, remember to make proper adjustments to the Solr schema, as [previously described](#setting-up-a-single-node-solrcloud-instance).
+
+```
+curl -X POST -H 'Content-type:application/json' --data-binary @src/main/resources/solr/schemas/SCHEMA_NAME.json http://localhost:8983/solr/COLLECTION_NAME/schema
+```
+
+Then, either *copy the index files to where Solr expected*, or *tell Solr where to look for*.
```

Comment:
Let's be opinionated - pick the better option (my vote is for the first) and do that - don't give the user the choice.